example_app_generator/generate_action_mailer_specs.rb

```diff
@@ -14,11 +14,25 @@ def comment_lines(path, flag, *args)
 
   initializer 'action_mailer.rb', <<-CODE
     if ENV['DEFAULT_URL']
-      Rails.application.configure do
-        config.action_mailer.default_url_options = { :host => ENV['DEFAULT_URL'] }
+      if ::Rails::VERSION::STRING < '4.1'
+        ExampleApp::Application.configure do
+          config.action_mailer.default_url_options = { :host => ENV['DEFAULT_URL'] }
+        end
+      else
+        Rails.application.configure do
+          config.action_mailer.default_url_options = { :host => ENV['DEFAULT_URL'] }
+        end
```

Comment:
Just curious why this change is related to this PR? Shouldn't we always have been doing this?